build-support/bin/generate_travis_yml.py

```diff
@@ -202,7 +202,7 @@ def docker_run_travis_ci_image(command: str) -> str:
       # We include the lmdb_store to include a local process cache, so that hopefully we don't
       # need to re-run processes (particularly tests) which have already run.
       # TODO(#8041): Prune this directory before storing the cache.
-      '${HOME}/.cache/pants/lmdb_store',
+      #'${HOME}/.cache/pants/lmdb_store',
```

Comment:
Why delete this in this PR? Although I agree we should not cache it, this seems better for a dedicated PR.